esmvaltool/cmorizers/data/formatters/datasets/esacci_lst.py

```diff
@@ -1,160 +1,138 @@
-
-    time_point = (datetime.datetime(year, month, 1, 0, 0) -
-                  datetime.datetime(1981, 1, 1, 0, 0, 0)).total_seconds()
-    monthly_co_time.points = time_point
-
-    num_days = monthrange(year, month)[1]
-    monthly_co_time.bounds = [
-        time_point, time_point + ((num_days - 1) * 24 * 3600)
-    ]
-    # should this be num_days or num_days-1 ### question for Valeriu or Axel
-    # or 23:59:59 ???
-
-    monthly_cube.attributes = {
-        'information': 'Mean of Day and Night Aqua MODIS monthly LST'
-    }
-
-    return monthly_cube
+                try:
+                    cubes = load_cubes(in_dir,
+                                       vals['file'],
+                                       year,
+                                       month,
+                                       vals['raw'],
+                                       )
+                except:
+                    logger.info(f'Problem: Month %s in %s' % (month, year))
+                    continue
+
+                #  make time coords
+                time_units = 'hours since 1970-01-01 00:00:00'
+                time_point = unit.date2num(datetime.datetime(year, month, 1),
+                                           time_units,
+                                           unit.CALENDAR_STANDARD
+                                           )
+
+                time_coord = iris.coords.DimCoord(time_point,
+                                                  standard_name='time',
+                                                  long_name='time',
+                                                  var_name='time',
+                                                  units=time_units,
+                                                  bounds=None,
+                                                  attributes=None,
+                                                  coord_system=None,
+                                                  circular=False
+                                                  )
+
+                cubes.attributes = {}
+                cubes.attributes['var'] = var
+
+                try:
+                    cubes.remove_coord('time')
+                except:
+                    logger.info('Coord fix issue %s' % cubes.long_name)
+
+                cubes.add_dim_coord(time_coord, 0)
+
+                if cubes.long_name == 'land surface temperature':
+                    cubes.long_name = 'surface_temperature'
+                    cubes.standard_name = 'surface_temperature'
+
+                try:
+                    cubes = fix_coords(cubes)
+                except:
+                    logger.info('skip fixing')
+                    logger.info(cubes.long_name)
+
+                # this is needed for V1 data, V3 data is ok
+                try:
+                    cubes.coords()[2].standard_name = 'longitude'
+                except:
+                    # No change needed
+                    pass
```

Comment:
what's the point of this? The code doesn't fail unless specifically instructed, and if it doesn't fail, and no warning or info is fed to the user then there is no point in having the check